implementations/rust/ockam/ockam_vault/src/software/legacy/secret.rs

```diff
@@ -23,24 +25,14 @@ impl<'de> Deserialize<'de> for Secret {
         D: Deserializer<'de>,
     {
         #[derive(Deserialize)]
-        pub enum SecretV1 {
-            Key(#[serde(with = "hex_encoding")] SecretKeyVec),
-            Aws(KeyId),
-        }
-        #[derive(Deserialize)]
-        struct SecretV2(#[serde(with = "hex_encoding")] SecretKeyVec);
+        struct SecretV2(#[serde(with = "hex_encoding")] Vec<u8>);
```

Comment:
I didn't want to mess with it, so just removed the old type so that it compiles. All of it will go away since we don't want to store secrets in the file at all